pdf_build_src/README.md

```diff
@@ -1,41 +1,47 @@
-# pdf-version of BIDS specification 
+# pdf-version of BIDS specification
 
-The `pdf_build_src` directory contains the scripts and tex files required to build a pdf document of the BIDS specification from multiple markdown files using the pandoc library. 
+The `pdf_build_src` directory contains the scripts and `.tex` files required to build a pdf document of the BIDS specification from multiple markdown files using the pandoc library.
 
-Pandoc is command line tool which is also a Haskell library that converts files from one markup format to another. More here: https://pandoc.org/index.html
+[Pandoc](https://pandoc.org/index.html) is command line tool which is also a Haskell library that converts files from one markup format to another.
 
 ## Requirements
 
-For the pdf build to be successful, the following need to be installed: 
+For the pdf build to be successful, the following need to be installed:
 
-- Python 3.x 
-- pandoc 
-- Latest version of LaTeX: By default, Pandoc creates PDFs using LaTeX. Because a full MacTeX installation uses four gigabytes of disk space, pandoc recommends BasicTeX or TinyTeX and using the tlmgr tool to install additional packages as needed. 
+- Python 3.6 or higher
+- Numpy
+- pandoc
+- Latest version of LaTeX: By default, Pandoc creates PDFs using LaTeX.
+  Because a full MacTeX installation uses four gigabytes of disk space,
+  pandoc recommends BasicTeX or TinyTeX and using the tlmgr tool to install additional packages as needed.
+- The [Symbola font](https://packages.ubuntu.com/search?keywords=fonts-symbola) that provides all emoji unicodes necessary to render the contributior icons in the PDF
```

Comment:
small typo: 
> contribut**ior** icons

Reply:
did you know that you can make code suggestions using this button (screenshot, see top right): 

![image](https://user-images.githubusercontent.com/9084751/89809119-6a3a9900-db3b-11ea-8211-c2b274920da2.png)

this will then open a markdown code formatting block with a "suggestion" markup.

If you edit in this code block and then submit your review comment, it becomes more easy to inspect or even to integrate in the PR :-) --> similar to how I kept bothering you in the table-PR

Reply:
```suggestion
- The [Symbola font](https://packages.ubuntu.com/search?keywords=fonts-symbola) that provides all emoji unicodes necessary to render the contributor icons in the PDF
```